### ISSUE TITLE
Can the use of models other than LLAMA be supported?

### ISSUE THREAD

**Issue by ojipadeson** [HuangLK/llama-deepspeed#45]
Is it possible to replace ```llama_pipeline_model.py``` with some other models such as ```qwen2_pipeline_model.py```. Btw, I would like to ask what the code of this pipeline model was generated with reference to. Thank you so much！

**Comment by HuangLK** [comment]
> Is it possible to replace llama_pipeline_model.py with some other models such as qwen2_pipeline_model.py. 

yes! qwen2 is quite similar with llama and it may not take much effort. 

>  I would like to ask what the code of this pipeline model was generated with reference to

basiclly refer  to [deepspeed doc](https://www.deepspeed.ai/tutorials/pipeline/#load-balancing-pipeline-modules)

However, sorry for the suspension of this project. TT
